src/Providers/AddonServiceProvider.php

```diff
@@ -11,32 +11,35 @@
 use Statamic\Exceptions\NotBootedException;
 use Statamic\Extend\Manifest;
 use Statamic\Facades\Addon;
+use Statamic\Facades\Fieldset;
+use Statamic\Facades\File;
 use Statamic\Statamic;
 use Statamic\Support\Str;
 
 abstract class AddonServiceProvider extends ServiceProvider
 {
-    protected $listen = [];
-    protected $subscribe = [];
-    protected $tags = [];
-    protected $scopes = [];
     protected $actions = [];
+    protected $commands = [];
+    protected $externalScripts = [];
+    protected $externalStylesheets = [];
     protected $fieldtypes = [];
+    protected $listen = [];
+    protected $middlewareGroups = [];
     protected $modifiers = [];
-    protected $widgets = [];
     protected $policies = [];
-    protected $commands = [];
-    protected $stylesheets = [];
-    protected $externalStylesheets = [];
-    protected $scripts = [];
-    protected $externalScripts = [];
     protected $publishables = [];
     protected $routes = [];
-    protected $middlewareGroups = [];
+    protected $scopes = [];
+    protected $scripts = [];
+    protected $stylesheets = [];
+    protected $subscribe = [];
+    protected $tags = [];
     protected $updateScripts = [];
     protected $viewNamespace;
-    protected $publishAfterInstall = true;
+    protected $widgets = [];
+
     protected $config = true;
+    protected $publishAfterInstall = true;
     protected $translations = true;
```

Comment:
Please revert the re-ordering of these properties. Just include any new or removed properties.